index.js

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2019 Ultimaker B.V.
 
+/* eslint-disable quote-props */
```

Comment:
If we always want to quote props we should maybe add that rule…

Reply:
We actually don't, I just do for this file because of all of ESLint's weird prop names with `-` and `/` in them (which we don't usually do)